app/src/main/java/org/fossasia/pslab/activity/MainActivity.java

```diff
@@ -57,6 +58,8 @@
     View navHeader;
     private ImageView imgProfile;
     private TextView txtName;
+    /**** CustomTabService*/
+    private CustomTabService cts;
```

Comment:
Please rename the variable and give name that makes some sense like `customTabService` or `issueTrackerTab`.